alpha/alpha-server/src/main/resources/log4j2.xml

```diff
@@ -16,25 +16,26 @@
   ~ limitations under the License.
   -->
 
-<Configuration status="WARN" monitorInterval="30">
+<Configuration status="WARN" monitorInterval="30" shutdownHook="disable">
   <Properties>
     <Property name="LOG_PATTERN">
       %d{yyyy-MM-dd HH:mm:ss.SSS} %5p ${hostName} --- [%15.15t] %-40.40c{1.} : %m%n%ex
     </Property>
+    <Property name="log-file-name">alpha-server</Property>
   </Properties>
   <Appenders>
     <Console name="Console" target="SYSTEM_OUT">
       <PatternLayout>
         <Pattern>${LOG_PATTERN}</Pattern>
       </PatternLayout>
     </Console>
-    <RollingFile name="FileAppender" fileName="logs/alpha-server.log"
-      filePattern="logs/alpha-server-%d{yyyy-MM-dd}-%i.log">
+    <RollingFile name="FileAppender" fileName="logs/${sys:log-file-name}.log"
```

Comment:
Why did you change the log file name?

Reply:
> Why did you change the log file name?

Sometimes we need to start multiple processes in the same location, so we can distinguish by the port number in the log name.

Reply:
The log file name is alpha-server-${server.port}.log. looks like  alpha-server-8080.log